src/plugins/charts/bar/inspector/BarGraphOptions.vue

```diff
@@ -316,11 +316,16 @@ export default {
                     }
                 } else {
                     if (this.yKey === undefined) {
-                        yKeyOptionIndex = this.yKeyOptions.findIndex((option, index) => index !== xKeyOptionIndex);
-                        if (yKeyOptionIndex > -1) {
+                        if (metadataValues.length && metadataArrayValues.length === 0) {
                             update = true;
-                            this.yKey = this.yKeyOptions[yKeyOptionIndex].value;
-                            this.yKeyLabel = this.yKeyOptions[yKeyOptionIndex].name;
+                            this.yKey = 'none';
```

Comment:
not following this code exactly, but it looks like we're setting the options to `none` if there aren't any options in the metadata?

Reply:
That's correct.